blocks/identity-block/features/header-account-action/_children/DropDownLinkListItem.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import getThemeStyle from 'fusion:themes';
+import { useFusionContext } from 'fusion:context';
+
+import './DropDownLinkListItem.scss';
+
+const StyledLinkWithHover = styled.a`
+  font-family: ${({ arcSite }) => getThemeStyle(arcSite)['primary-font-family']};
+  color: ${(arcSite) => getThemeStyle(arcSite)['primary-color']};
+
+  &:hover {
+    color: ${(arcSite) => getThemeStyle(arcSite)['primary-color']};
```

Comment:
Should we be consistent and pass in the values for primary-color and font-family?

Reply:
yep will do

Reply:
updated